Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/pull_request_unittest.py

```diff
@@ -720,6 +720,67 @@ def test_github_reopen(self):
             ],
         )
 
+    def test_github_substring_branch(self):
+        # Test that a branch that is a substring of another branch doesn't match
+        # For example, 'eng/Adopt-LIFETIME_BOUND-for-WTF-Ref' should not match
+        # a closed PR with branch 'eng/Adopt-LIFETIME_BOUND-for-WTF-RefPtr'
+        with mocks.remote.GitHub() as remote, mocks.local.Git(
+            self.path, remote='https://{}'.format(remote.remote),
+            remotes=dict(fork='https://{}/Contributor/WebKit'.format(remote.hosts[0])),
+        ) as repo, mocks.local.Svn(), patch('webkitbugspy.Tracker._trackers', []):
+            # First create a PR with the longer branch name
+            with OutputCapture():
```

Comment:
There shouldn't be any output to even capture by using `repo.commits` (postulating without checking the code). Anyhow, it'd be better to move this up into the `with` statement above and delete the second one at line 756.